helm/rpm-builder/values.yaml

```diff
@@ -0,0 +1,28 @@
+targetNamespace: rfe
+
+buildConfig:
+  name: rpm-builder
```

Comment:
I think that since this chart is not part of the default deployment of the RFE, this image name might be too generic. No? (same for the output image name)

Reply:
The `buildconfig` generates an image that contains all the dependencies to build and package rpms, so the name is generic enough to contain the purpose of the image. I didn't see any reason to name it otherwise when there are no specifics to our project included.